google/cloud/sql/connector/__init__.py

```diff
@@ -17,7 +17,7 @@
 from typing import List
 
 from .connector import connect, Connector
-from .instance_connection_manager import IPTypes
+from .instance import IPTypes
```

Comment:
@kurtisvg is this a breaking change?

Reply:
Good question. I don't think so, because I _think_ the user still imports it from the same module ( `from google.cloud.sql.connector import IPTypes`)

However, our documentation doesn't show this. Can you update it to include the imports: https://github.com/GoogleCloudPlatform/cloud-sql-python-connector#specifying-public-or-private-ip

Reply:
Created PR #317 to address this